src/main/java/org/rundeck/client/tool/options/JobBaseOptions.java

```diff
@@ -22,6 +22,8 @@
 
 public interface JobBaseOptions extends ProjectNameOptions {
 
+    /* Moved to JobFileOptions
```

Comment:
Instead of commenting this out, can just remove this class since it has no use now

Reply:
done